ecowitt2mqtt/rootfs/etc/services.d/ecowitt2mqtt/run

```diff
@@ -112,8 +120,8 @@ bashio::log.info "Starting Ecowitt2MQTT"
 exec ecowitt2mqtt \
     --mqtt-broker="${MQTT_HOST}" \
     --mqtt-port="${MQTT_PORT}" \
-    --mqtt-username="${MQTT_USERNAME}" \
-    --mqtt-password="${MQTT_PASSWORD}" \
+    ${MQTT_USER_ARG} \
+    ${MQTT_PASS_ARG} \
```

Comment:
Previous line will evaluate to an empty string in case no MQTT_PASSWORD will be provided; the same applies for the line 123. Or at least this is what I intend to do here :-)